build.gradle

```diff
@@ -39,15 +39,19 @@ pluginBundle {
 github {
     repositoryName = "wooga/atlas-github-release-notes"
 }
-
 dependencies {
-    testCompile('org.spockframework:spock-core:1.2-groovy-2.4') {
-        exclude module: 'groovy-all'
-    }
-
-    testCompile 'com.github.stefanbirkner:system-rules:1.18.0'
-    testCompile "org.ajoberstar:gradle-git:1.7.2"
-    testCompile 'com.wooga.spock.extensions:spock-github-extension:0.1.0'
+    testCompile "org.ajoberstar:grgit:1.7.2"
+    testCompile'com.wooga.spock.extensions:spock-github-extension:0.1.2'
     compile 'com.wooga.github:github-changelog-lib:0.4.+'
-    compile "gradle.plugin.net.wooga.gradle:atlas-github:1.+"
+    compile "gradle.plugin.net.wooga.gradle:atlas-github:2.+"
 }
+configurations.all {
+    resolutionStrategy {
+        force 'org.codehaus.groovy:groovy-all:2.5.12'
+        force 'org.codehaus.groovy:groovy-macro:2.5.12'
+        force 'org.codehaus.groovy:groovy-nio:2.5.12'
+        force 'org.codehaus.groovy:groovy-sql:2.5.12'
+        force 'org.codehaus.groovy:groovy-xml:2.5.12'
+    }
+}
```

Comment:
So this is a hardcore override for the these specific depenendencies. As long as it works ;)

Reply:
Yeah, its working, and it sounds better for me than go and try guessing which libs are with the wrong dependencies and ignoring them one by one. However, I don't know if there any nasty long-term side effects for this. However, it is working fine now.